Week3/homework/step2-2.js

```diff
@@ -3,18 +3,27 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
-  // Replace this comment and the next line with your code
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+  for (let i = startIndex; i <= stopIndex; i++) {
```

Comment:
Although your code passes the test, you were asked to iterate through the `numbers` array you created in line 6-7. Can you please correct this?

Reply:
I am not sure, but I am actually using a for loop here to iterate through the 'numbers' array.

Reply:
But you are not using the `numbers` array. The variable `numbers` is not referenced at all in your second loop. I would expect something like:

```js
for (let i = 0; i < numbers.length; i++) {
  const number = numbers[i];
  if (number % 3 === 0) {
    ...
```

Reply:
My bad...It's done.
Thanks!!